sklearn/base.py

```diff
@@ -562,3 +561,37 @@ def is_regressor(estimator):
         True if estimator is a regressor and False otherwise.
     """
     return getattr(estimator, "_estimator_type", None) == "regressor"
+
+
+def is_pairwise(estimator):
```

Comment:
I'm not 100% sure if adding new functions is the way to go if we're gonna move towards estimator tags. @jnothman what's your opinion? From the docstring, the distinction between the two functions is not entirely clear.